src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -1651,7 +1695,7 @@ public void Method()
   }
 }
 ";
-            await VerifyItemExistsAsync(markup, "vs");
+            await VerifyItemExistsAsync(markup, "immutableArray");
```

Comment:
Both the old and the new behavior are incorrect here. There is tracked by #36364.